ci/build_container/build_recipes/zlib.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+VERSION=1.2.11
+
+wget -O zlib-$VERSION.tar.gz https://github.com/madler/zlib/archive/v$VERSION.tar.gz
+tar xf zlib-$VERSION.tar.gz
+cd zlib-$VERSION
+./configure
+make V=1 install
+cp libz.a $THIRDPARTY_BUILD/lib
```

Comment:
Are these 2 lines necessary? Can we just set --prefix, etc. on ./configure like the other recipes are doing?